requirments.txt

```diff
@@ -1,4 +1,6 @@
-py==1.4.34
-pytest==3.1.3
 apache-libcloud==2.1.0
+class-registry==2.1.0
 paramiko==2.2.1
+py==1.4.34
```

Comment:
what's that? You don't seem to use it

Reply:
It's pytest dependency. But I'll delete it from requirments.txt